packages/nodes-base/nodes/Google/CloudStorage/BucketDescription.ts

```diff
@@ -0,0 +1,617 @@
+	const body = this.getNodeParameter('createBody') as IDataObject;
+
+	// Parse all the JSON fields
+	if (body.acl) {
+		try {
+			body.acl = JSON.parse(body.acl as string);
+		} catch (error) {}
+	}
+	if (body.billing) {
+		try {
+			body.billing = JSON.parse(body.billing as string);
+		} catch (error) {}
+	}
+	if (body.cors) {
+		try {
+			body.cors = JSON.parse(body.cors as string);
+		} catch (error) {}
+	}
+	if (body.customPlacementConfig) {
+		try {
+			body.customPlacementConfig = JSON.parse(body.customPlacementConfig as string);
+		} catch (error) {}
+	}
+	if (body.dataLocations) {
+		try {
+			body.dataLocations = JSON.parse(body.dataLocations as string);
+		} catch (error) {}
+	}
+	if (body.defaultObjectAcl) {
+		try {
+			body.defaultObjectAcl = JSON.parse(body.defaultObjectAcl as string);
+		} catch (error) {}
+	}
+	if (body.encryption) {
+		try {
+			body.encryption = JSON.parse(body.encryption as string);
+		} catch (error) {}
+	}
+	if (body.iamConfiguration) {
+		try {
+			body.iamConfiguration = JSON.parse(body.iamConfiguration as string);
+		} catch (error) {}
+	}
+	if (body.labels) {
+		try {
+			body.labels = JSON.parse(body.labels as string);
+		} catch (error) {}
+	}
+	if (body.lifecycle) {
+		try {
+			body.lifecycle = JSON.parse(body.lifecycle as string);
+		} catch (error) {}
+	}
+	if (body.logging) {
+		try {
+			body.logging = JSON.parse(body.logging as string);
+		} catch (error) {}
+	}
+	if (body.retentionPolicy) {
+		try {
+			body.retentionPolicy = JSON.parse(body.retentionPolicy as string);
+		} catch (error) {}
+	}
+	if (body.versioning) {
+		try {
+			body.versioning = JSON.parse(body.versioning as string);
+		} catch (error) {}
+	}
+	if (body.website) {
+		try {
+			body.website = JSON.parse(body.website as string);
+		} catch (error) {}
+	}
+
+	requestOptions.body = Object.assign(requestOptions.body, body);
+	return requestOptions;
+}
+
+export const bucketOperations: INodeProperties[] = [
+	{
+		displayName: 'Operation',
+		name: 'operation',
+		type: 'options',
+		noDataExpression: true,
+		displayOptions: {
+			show: {
+				resource: ['bucket'],
+			},
+		},
+		options: [
+			{
+				name: 'Create',
+				value: 'create',
+				description: 'Create a new Bucket',
+				routing: {
+					request: {
+						method: 'POST',
+						url: '/b/',
+						qs: {
+							project: '={{$parameter["projectId"]}}',
+							projection: '={{$parameter["projection"]}}',
+						},
+						body: {
+							name: '={{$parameter["bucketName"]}}',
+						},
+						returnFullResponse: true,
+					},
+					send: {
+						preSend: [
+							async function (this, requestOptions) {
+								if (!requestOptions.qs) requestOptions.qs = {};
+								const additionalQs = this.getNodeParameter('createAcl') as IDataObject;
+
+								// Merge in the options into the queryset
+								requestOptions.qs = Object.assign(requestOptions.qs, additionalQs);
+								return requestOptions;
+							},
+						],
+					},
```

Comment:
This whole block can be removed with the addition of the routing suggested below.